tests/unit/conditions/test_json_api_condition.py

```diff
@@ -258,6 +258,27 @@ def test_json_api_condition_evaluation_with_auth_token(mocker):
     )
 
 
+def test_json_api_condition_evaluation_with_user_address_context_variable(
+    mocker, valid_eip712_auth_message
```

Comment:
Let's use `valid_eip4361_auth_message` instead. EIP712 support will be deprecated in the future. It hasn't yet because older versions of `@nucypher/taco` (`taco-web`) used it.

Reply:
I thought I'd found a bug, but this test proves that I haven't.
More than happy to make the switch to 4361

Reply:
> I thought I'd found a bug, but this test proves that I haven't.

😌 - that's a relief! Thanks for the additional testing. I'm always pro more tests.

Side note: you can add a dev newsfragment for this PR to make the "Check Release Note" CI job pass.